testing/test_helpconfig.py

```diff
@@ -38,6 +38,35 @@ def test_help(testdir):
     )
 
 
+def test_none_help_param_raises_exception(testdir):
+    """Tests a None help param raises a TypeError.
+    """
+    testdir.makeconftest(
+        """
+        def pytest_addoption(parser):
+            parser.addini("test_ini", None, default=True, type="bool")
+    """
+    )
+    result = testdir.runpytest("--help")
+    result.stderr.fnmatch_lines(
+        ["*TypeError: help argument cannot be None for test_ini*"]
+    )
+
+
+def test_empty_help_param(testdir):
+    """Tests an empty help param is displayed correctly.
+    """
+    testdir.makeconftest(
+        """
+        def pytest_addoption(parser):
+            parser.addini("test_ini", "", default=True, type="bool")
+    """
+    )
+    result = testdir.runpytest("--help")
+    assert result.ret == 0
+    result.stdout.fnmatch_lines(["*test_ini*(bool):*"])
```

Comment:
This doesn't really test that the help is *displayed* correctly. I suggest:

- Include a few more surrounding lines
- Pass `conescutive=True` to `fnmatch_lines`
- Avoid using the leading `*`, in order to test the indentation

Reply:
Thanks for the feedback! I've updated the PR, let me know what you think.